src/uiTest/scala/com/virtuslab/gitmachete/uitest/RunningIntelliJFixtureExtension.scala

```diff
@@ -6,22 +6,10 @@ import org.intellij.lang.annotations.Language
 import org.junit.Assert
 import org.virtuslab.ideprobe.{IdeProbeFixture, ProbeDriver, RunningIntelliJFixture}
 import org.virtuslab.ideprobe.Extensions._
-import org.virtuslab.ideprobe.dependencies.Plugin
 import org.virtuslab.ideprobe.robot.RobotPluginExtension
 
 trait RunningIntelliJFixtureExtension extends RobotPluginExtension { this: IdeProbeFixture =>
 
-  private val machetePlugin: Plugin = {
-    val pluginPath = sys.props
-      .get("ui-test.plugin.path")
-      .filterNot(_.isEmpty)
-      .getOrElse(throw new Exception("Plugin path is not provided"))
-    Plugin.Direct(Paths.get(pluginPath).toUri)
-  }
-
-  registerFixtureTransformer(_.withPlugin(machetePlugin))
-  registerFixtureTransformer(_.withAfterIntelliJStartup((_, intelliJ) => intelliJ.ide.configure()))
```

Comment:
Can't be done via config; moved to `UISuite` tho, as this trait doesn't look like a good place for side-effecting statements